mongo/mongo.go

```diff
@@ -83,3 +83,40 @@ func (m *Mongo) GetRole(ctx context.Context, id string) (*models.Role, error) {
 
 	return &role, nil
 }
+
+// GetRoles retrieves all role documents from Mongo, according to the provided limit and offset.
+//Offset and limit need to  be positive or zero. Zero limit is equivalent to no limit and all items starting at the offset will be returned.
+func (m *Mongo) GetRoles(ctx context.Context, offset, limit int) (*models.Roles, error) {
+	if offset < 0 || limit < 0 {
+		return nil, apierrors.ErrLimitAndOffset
+	}
+
+	s := m.Session.Copy()
+	defer s.Close()
+	log.Event(ctx, "getting roles", log.INFO)
```

Comment:
This message is kind of vague and not very helpful. For example if you were investigating a defect or live issue this log event is not very clear about what the context is in - where is it getting them from? Is this an API log message or a message form DB? 
Something like `querying document store for list of roles` gives more context - if you see that message you have a precise area of the codebase to look into.

Sorry to be pedantic but clear log messages with helpful details are really valuable when investigating issues but are easy to overlook.